gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -933,33 +933,151 @@ gst_tensor_transform_finalize (GObject * object)
   G_OBJECT_CLASS (parent_class)->finalize (object);
 }
 
+/**
+ * @brief Dimension conversion calculation
+ * @param[in] filter "this" pointer
+ * @param[in] direction GST_PAD_SINK if input->output conv
+ * @param[in] idx index of the input tensors
+ * @param[in] in_info tensor info structure of source tensor (input if direction is SINK)
+ * @param[out] out_info tensor info structure of destination tensor (output if direction is SINK)
+ * @return TRUE if success
+ */
+static gboolean
+gst_tensor_transform_convert_dimension (GstTensorTransform * filter,
```

Comment:
Can you keep the position of this function and only add function declaration to reduce changes? (making reviews easier)

Reply:
updated